cmd/bot/config.yaml

```
@@ -0,0 +1,18 @@
+database:
+    connectionuri: ?cache=shared&mode=rw
+    dbname: telebot.db
+    driver: sqlite3
+    timeout: 2
+debug: true
+hostname: tgx.egais-help.ru
+htmlport:  "3550"
```

YAMLlint (1.37.1)

[warning] too many spaces after colon

(colons)